applications/machine_learning/configuration/nrf5340dk_nrf5340_cpuapp/prj.conf

```diff
@@ -131,7 +131,7 @@ CONFIG_BT_BONDABLE=n
 
 CONFIG_BT_PERIPHERAL=y
 CONFIG_BT_PERIPHERAL_PREF_MIN_INT=6
-CONFIG_BT_PERIPHERAL_PREF_MAX_INT=6
+CONFIG_BT_PERIPHERAL_PREF_MAX_INT=9
```

Comment:
only this change here?

Reply:
The nRF53 DK forwards the data over UART. We cannot apply changes related to data forwarder over BLE NUS. There is no need to increase number of Bluetooth TX packets too.

Bluetooth is used here only for DFU over BLE SMP (for consistency with Thingy:53).

Reply:
I wanted to make preferred connection parameters the same among boards, if needed I can change it.